app/models/event.rb

```diff
@@ -0,0 +1,47 @@
+class Event
+
+  attr_reader :name, :start_datetime, :end_datetime, :description, :location, :link, :thumbnail
+
+	def initialize(event_json)
+	  @name = event_json['summary']
+	  @start_datetime = event_json['start']
+	  @end_datetime = event_json['end']
+	  @location = event_json['location']
+	  @description = event_json['description']
+	  @link = (event_json['description'].to_s).split.first
+	  @thumbnail = (event_json['description'].to_s).split.second
+	end
+
+  def self.pull_calendar_data
+  	#Establish base uri for Google Calendar API
+	  uri = URI('https://www.googleapis.com/calendar/v3/calendars/calendarId/events')
```

Comment:
You don't seem to be using the Google API wrapper anymore. Any reasons? Wrapper gems usually keep code cleaner. The code right now is pretty cluttered for a simple HTTP GET request